### PR TITLE
PP: Define name_list once

### PR DESCRIPTION
**Pull request by fernandog**
@medariox ideas to the comment above the name_list ?

### CODE REVIEW

**Comment by fernandog** [comment]
oh sh*.... dict loses the order when converting to list @#$%¨&

**Comment by fernandog** [comment]
Its failing because of the wrong order in items_resource

@medariox do you think is worth to use OrderedDict to keep this new logs?

File: medusa/post_processor.py [hunk 1]
```diff
@@ -87,7 +89,9 @@ def __init__(self, file_path, nzb_name=None, process_method=None, is_priority=No
 
         self.manually_searched = False
 
-        self.item_resources = {'file name': self.file_name, 'relative path': self.rel_path, 'nzb name': self.nzb_name}
+        self.item_resources = OrderedDict({'file name': self.file_name,
```
Comment (ratoaq2) [inline medusa/post_processor.py]: This way you have the same issue since you're using a regular dict to Instantiate an ordered one

**Comment by fernandog** [comment]
@medariox all good here: 

2017-02-07 09:46:39 DEBUG    FINDSUBTITLES :: [edfb56d] Show information parsed from file name

approve?

**Pull request merged**